newspaper/resources/text/stopwords-hi.txt

```diff
@@ -0,0 +1,43 @@
+को
```

Comment:
I am going to LGTM this and merge it in, my only comment is that the hindi stopwords list found on other sites seems larger: https://github.com/stopwords-iso/stopwords-hi/blob/master/stopwords-hi.txt

How did you determine that this is the best performing list?

Reply:
This seems to be a more comprehensible list!